src/plugins/indicator_libmessaging.py

```diff
@@ -1,14 +1,18 @@
+"""
+src/plugins/indicator_libmessaging.py
+=====================================
+"""
 # -*- coding: utf-8 -*-
-
 import gi
-gi.require_version('MessagingMenu', '1.0')  # noqa:E402
 from gi.repository import MessagingMenu
-
 from pybitmessage.bitmessageqt.utils import str_broadcast_subscribers
 from pybitmessage.tr import _translate
 
+gi.require_version('MessagingMenu', '1.0')  # noqa:E402
```

Comment:
No way. This should go before the import.

Reply:
Okay, will update the code. 
thank you

Reply:
E402 "module level import not at top of file"
Getting this error, after importing gi.require_version('MessagingMenu', '1.0') before the import

Reply:
Because you've put it in the wrong place and broken suppressing comment. That line was in the right place before you started to edit the code you don't understand.

Reply:
I understand the importance of line before import, but I'm getting flake8 and pycodestyle error while writing that line before import. Can, you please suggest me, what should I do here?

Reply:
Do nothing. I don't see the point to edit this module. Moreover, it is difficult to debug a broken plugin.